packages/cli/commands/project/cloneApp.js

```diff
@@ -0,0 +1,177 @@
+const fs = require('fs');
+const path = require('path');
+const {
+  addAccountOptions,
+  addConfigOptions,
+  getAccountId,
+  addUseEnvironmentOptions,
+} = require('../../lib/commonOpts');
+const { trackCommandUsage } = require('../../lib/usageTracking');
+const { loadAndValidateOptions } = require('../../lib/validation');
+const { i18n } = require('../../lib/lang');
+const {
+  selectPublicAppPrompt,
+} = require('../../lib/prompts/selectPublicAppPrompt');
+const { promptUser } = require('../../lib/prompts/promptUtils');
+const { poll } = require('../../lib/polling');
+const {
+  uiLine,
+  uiCommandReference,
+  uiAccountDescription,
+} = require('../../lib/ui');
+const SpinniesManager = require('../../lib/ui/SpinniesManager');
+const {
+  logApiErrorInstance,
+  ApiErrorContext,
+} = require('../../lib/errorHandlers/apiErrors');
+const { EXIT_CODES } = require('../../lib/enums/exitCodes');
+const { isAppDeveloperAccount } = require('../../lib/accountTypes');
+const {
+  cloneApp,
+  checkCloneStatus,
+  downloadClonedProject,
+} = require('@hubspot/local-dev-lib/api/projects');
+const { getCwd, isValidPath } = require('@hubspot/local-dev-lib/path');
+const { logger } = require('@hubspot/local-dev-lib/logger');
+const { getAccountConfig } = require('@hubspot/local-dev-lib/config');
+const {
+  fetchPublicAppMetadata,
+} = require('@hubspot/local-dev-lib/api/appsDev');
+const { extractZipArchive } = require('@hubspot/local-dev-lib/archive');
+
+const i18nKey = 'commands.project.subcommands.cloneApp';
+
+exports.command = 'clone-app [--appId] [--location]';
```

Comment:
Do we want to include the positional arguments here?  I think the plan is to move away from this in favor of flags

Reply:
The dashes in front of these mark them as flags. Going to mark this as resolved.

Reply:
I noticed this weird behavior when I was working on the deploy command, they get treated as positional args when included here.  

![Screenshot 2024-07-18 at 8 59 36 AM](https://github.com/user-attachments/assets/f6cf8f4d-c32e-4703-bd81-04804f9f4f77)

Reply:
Looked over the `yargs` docs, and they don't even mention flags in relation to the `command` method: https://github.com/yargs/yargs/blob/main/docs/advanced.md#commands. I'll remove them. Thanks for catching this, Joe🥇